osi_sensordata.proto

```diff
@@ -79,6 +80,9 @@ message SensorData
     // \note Should not be used by subscribers to SensorData data. 
     // Generally this field should be cleared after internal processing.
     optional ModelInternal model_internal = 12;
+    
```

Comment:
I'm still somewhat confused by the need for this in SensorData: Since the input of the sensor model will not have any data in SensorData, this would not help the Sensor Model, rather it would fall to the sensor model to fill this information from GroundTruth information, and the consumer would be the recipient of sensor data, and I think those functions would rather have more driving dynamics information than is contained in GroundTruth? @CarloVanDriestenBMW ?

Reply:
The original idea to have a Ego Vehicle in SensorData is to model a GPS sensor. The noise from GPS influences the Ego Vehicle. If we do not have the Ego Vehicle in SensorData, then the noise of a GPS sensor would be added up to other sensors output. Then, a possible solution is to separate a new field in the SensorData to indicate the Ego Vehicle noise (offset), in this case a Ego Vehicle in SensorData is unnecessary. @CarloVanDriestenBMW

Reply:
I see, in that case I agree that adding a new object for sensed ego vehicle data makes more sense (sensor might not only rely on GPS, but e.g. map data and distances to landmarks, etc., and might also include sensed environment data, like e.g. visibility, etc.). Probably makes sense to introduce this as a new branch & pull request, so I can close this one?

Reply:
@pmai yes close that one and make a consolidated branch